packages/forma-36-react-components/src/components/Button/README.md

```diff
@@ -0,0 +1,117 @@
+# Button
```

Comment:
Adding a table of contents would be great, but maybe this can be autogenerated when we consume the readme in the Forma 36 docs website

Reply:
@m10l I think we might want to create some sort of secondary menu in the website? something that is sticky and user is able to use it when scrolling down? But it would be some sort of table of contents I guess...